packages/commerce-server/src/format-prices-for-product.ts

```diff
@@ -166,12 +166,22 @@ export async function formatPricesForProduct(
       purchaseToBeUpgraded: upgradeFromPurchase,
     })
 
-  const fixedDiscountForUpgrade = await getFixedDiscountForUpgrade({
-    purchaseToBeUpgraded: upgradeFromPurchase,
-    productToBePurchased: product,
-    purchaseWillBeRestricted: appliedCouponType === 'ppp',
-    ctx,
-  })
+  const fireFixedDiscountForIndividualUpgrade = async () => {
+    return await getFixedDiscountForIndividualUpgrade({
+      purchaseToBeUpgraded: upgradeFromPurchase,
+      productToBePurchased: product,
+      purchaseWillBeRestricted: appliedCouponType === 'ppp',
+      ctx,
+    })
+  }
+
+  // Right now, we have fixed discounts to apply to upgrades for indvidual
+  // purchases. If it is a bulk purchase, a fixed discount shouldn't be
+  // applied. It's likely this will change in the future, so this allows us
+  // to handle both and distinguishes them as two different flows.
+  const fixedDiscountForUpgrade = result.bulk
+    ? 0
+    : await fireFixedDiscountForIndividualUpgrade()
```

Comment:
Separate out the fixed discount based on the individual and bulk path. For now, a fixed discount on a bulk purchase is always going to be 0. It will be easy enough to insert some different functionality when that changes.

As for the existing fixed discount function, that has been renamed to `getFixedDiscountForIndividualUpgrade` to emphasize that it is for **individual** purchases, and shouldn't be used for bulk.